fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -293,13 +340,29 @@ if [ ! -e "$fMRIFolder" ] ; then
   log_Msg "mkdir ${fMRIFolder}"
   mkdir "$fMRIFolder"
 fi
-cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
+${FSLDIR}/bin/imcp "$fMRIScout" "$fMRIFolder"/"$OrigScoutName"
+
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction"
+    SliceTimerCorrectionParameters=`echo ${SliceTimerCorrectionParameters} | sed 's/@/ /g'` 
+    ${FSLDIR}/bin/slicetimer -i "$fMRIFolder"/"$OrigTCSName" -o "$fMRIFolder"/"$OrigTCSName"_orig_stc -r ${TR} -v ${SliceTimerCorrectionParameters}
+    
+    if [ "${DistortionCorrection}" = "NONE"] ; then {
+      #  use FSL's fslreorient2std for reorienting the image to match the approximate orientation of the standard template images MNI152
+      #  this makes the single-band processing more robust to registration problems.
+      #  NOTE: Take into account that BOLD images might have a different spatial orientation!
+
+      log_Warn "Performing fslreorient2std! Please take that into account when using volume BOLD images in further analyses!"
+      ${FSLDIR}/bin/fslreorient2std "$fMRIFolder"/"$OrigTCSName"_orig_stc "$fMRIFolder"/"$OrigTCSName"
```

Comment:
Probably >95% of fMRI is axially acquired, and thus will already be in LAS or RAS orientation.  Thus, doing this indiscriminately is not necessary.  Here is some code you can use to assess the orientation and proceed accordingly:

```
xorient=`$FSLDIR/bin/fslval "$fMRIFolder"/"$OrigTCSName" qform_xorient | tr -d ' '`
yorient=`$FSLDIR/bin/fslval "$fMRIFolder"/"$OrigTCSName" qform_yorient | tr -d ' '`
zorient=`$FSLDIR/bin/fslval "$fMRIFolder"/"$OrigTCSName" qform_zorient | tr -d ' '`

log_Msg "$fMRIFolder/$OrigTCSName: xorient=${xorient}, yorient=${yorient}, zorient=${zorient}"

if [[ "$xorient" != "Right-to-Left" && "$xorient" != "Left-to-Right" || \
      "$yorient" != "Posterior-to-Anterior" || \
      "$zorient" != "Inferior-to-Superior" ]] ; then
reorient=TRUE
else 
reorient=FALSE
fi

```

Reply:
Even better flow:
Start with my proposed comments
Use code to set the reorient value (TRUE or FALSE)
Then have a single conditional: `if [[ $reorient = "TRUE" && $DistortionCorrection = "NONE" ]] ; then`

Reply:
Ok. I hope I got it right.